client/src/containers/FormAndEventsWrapper.js

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
 
-import Events from './Events';
-import EventsSource from './EventsSource';
+import Events from '../components/Events';
+import EventsSource from '../components/EventsSource';
 
 class FormAndEventsWrapper extends React.Component {
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.state = { value: '' };
+    this.handleSelection = this.handleSelection.bind(this);
+    this.state = { value: '',
```

Comment:
might be nice to move { value: '', selection: 'beer' } to a `const` called `defaultState`, and do `this.state = defaultState;` here, but it's probably alright like this, no biggie.

Reply:
clue is in the fact it's called `constructor` after all